Content.Server/Silicons/Laws/SiliconLawSystem.cs

```diff
@@ -295,6 +295,12 @@ protected override void OnUpdaterInsert(Entity<SiliconLawUpdaterComponent> ent,
         while (query.MoveNext(out var update))
         {
             SetLaws(lawset, update);
+            if (provider.LawUploadSound != null)
+            {
+                if (!_mind.TryGetMind(update, out var mindId, out _))
+                    return;
+                _roles.MindPlaySound(mindId, provider.LawUploadSound);
+            }
```

Comment:
```suggestion
            if (provider.LawUploadSound != null && _mind.TryGetMind(update, out var mindId, out _) )
                _roles.MindPlaySound(mindId, provider.LawUploadSound);
```
you can combine the two if conditions